demo/components/demo/components/notifications/notifications.html

```diff
@@ -13,4 +13,9 @@ <h2 class="slds-text-heading--small">This is a toast with an icon next to its me
   </div>
 </ngl-notification>
 
-<button type="button" nglButton="destructive" class="slds-m-top--large" (click)="showAlert = true;">Show alert</button>
+<button type="button" nglButton="destructive" class="slds-m-top--large" (click)="show()">Show alert</button>
+
+<div class="slds-m-top--large">
+  After 5 seconds, the alert will be automatically closed, if the <strong>X</strong> close button is not clicked.
```

Comment:
You should add a `timeout` usage in the readme, clarifying that any time the input value changes a new timeout is set (if this is the case) and the previous one is canceled. This way you can keep the demo much simpler, which is the target.